### DISCUSSION TITLE
Adding axis text at the end of the axis

### DISCUSSION THREAD

**Discussion by plunavat** [jfree/jfreechart#391] [Q&A]
Hello,

We need to show units at the end of the axis as below

![image](https://github.com/jfree/jfreechart/assets/5730057/05966ec5-8ac3-48b0-b474-e51e1e0dea7f)

As shown at the right side of the axis "Q/m3/hr"

How do we achieve the same using jfreechart. Currently we are showing as axis label, but it is shown below the axis

![image](https://github.com/jfree/jfreechart/assets/5730057/017bcedb-5cab-47d6-b920-df97e0f1f054)

Thanks in advance

Regards,
Pranav

**Comment by trashgod** [comment]
I'm not aware of any support for this, but I'd welcome correction. Alternatively, you might consider an `AttributedString` for the axis label, seen [here](https://stackoverflow.com/a/29808858/230513), [here](https://stackoverflow.com/a/46007513/230513), and [here](https://stackoverflow.com/a/76585316/230513).

**Reply by plunavat** [reply]
Thanks @trashgod for your reply.
But I need to display text (m3/hr) at the end of the domain axis after the last tick (Refer image 1)

Regards,
Pranav

**Reply by jmuthusa** [reply]
Hi, Looking for similar solution to add text at the end of the axes as in the image below.
<img width="365" alt="image" src="https://github.com/user-attachments/assets/02d76a38-461a-4529-ae93-8e758af874d4" />
Let know if there is a workaround.
Thanks in advance.
